uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/directives/leave-request-popup.js

```diff
@@ -34,12 +34,18 @@ define([
           contactId: '<',
           leaveRequest: '<',
           leaveType: '@',
-          forStaff: '<'
+          userRole: '@'
         },
         restrict: 'EA',
         link: function (scope, element) {
           var controller = _.capitalize(getLeaveType(scope.leaveType, scope.leaveRequest)) + 'RequestCtrl';
 
+          if (!scope.userRole) {
+            $log.error("User role not specified.");
```

Comment:
although i think it would make more sense to set the default fallback in the controller, instead of relying on the directive to give the controller the default